packages/devtools_app/lib/src/screens/inspector_v2/widget_properties/properties_view.dart

```diff
@@ -20,26 +24,70 @@ class PropertiesView extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ValueListenableBuilder<List<RemoteDiagnosticsNode>>(
-      valueListenable: controller.selectedNodeProperties,
-      builder: (context, properties, _) {
-        return Container(
-          margin: const EdgeInsets.all(denseSpacing),
-          child: ListView.builder(
-            itemCount: properties.length,
-            itemBuilder: (context, index) {
-              return PropertyItem(
-                index: index,
-                properties: properties,
-              );
-            },
+    return MultiValueListenableBuilder(
+      listenables: [
+        controller.selectedNodeWidgetProperties,
+        controller.selectedNodeRenderProperties,
+      ],
+      builder: (context, values, _) {
+        final widgetProperties = values.first as List<RemoteDiagnosticsNode>;
+        final renderProperties = values.second as List<RemoteDiagnosticsNode>;
+        return Padding(
+          padding: const EdgeInsets.all(denseSpacing),
+          child: AnalyticsTabbedView(
+            gaScreen: gac.inspector,
+            tabs: [
+              (
+                tab: DevToolsTab.create(
+                  tabName: 'Widget properties',
+                  gaPrefix: 'widgetPropertiesTab',
```

Comment:
this should be the same as the gaPrefix used below.  The prefix is affixed to the tab name to create the gaId for the tab. So maybe 'inspectorNodeProperties' or something like that. You can put this in a const for shared use between the two DevToolsTab objects

Reply:
Done!